kedro-datasets/kedro_datasets/video/video_dataset.py

```diff
@@ -67,10 +68,11 @@ def __getitem__(self, index: Union[int, slice]):
 class FileVideo(AbstractVideo):
     """A video object read from a file"""
 
-    def __init__(self, filepath: str) -> None:
+    def __init__(self, filepath: str, metadata: Dict[str, Any] = None) -> None:
```

Comment:
Same here. The only dataset in this file is the `VideoDataSet`, so that's the only class where the `metadata` should be added.